src/main/java/org/galatea/starter/domain/IexPriceID.java

```diff
@@ -0,0 +1,17 @@
+package org.galatea.starter.domain;
+
+import java.io.Serializable;
+import lombok.Data;
+
+@Data
```

Comment:
Modelling this way means you end up storing the date and symbol twice.

Better would be a db object like this: 
{
key:{keyData},
otherData:value,
}

that you persist, and you flatten it to remove db-specific data for the consumer

Reply:
essentially having 2 representations, one for persistence, and one for the client